federatedscope/db/processor/local_processor.py

```diff
@@ -1,43 +1,40 @@
 from federatedscope.db.processor.basic_processor import BasicSQLProcessor
+from federatedscope.db.model.sqlschedule_pb2 import DataType
 from federatedscope.db.model.backup import SQLQuery, Filter, Aggregate
 from federatedscope.db.data.schema import Attribute, Schema
 from federatedscope.db.data.data import Data
 
+
 import pandas as pd
+import numpy as np
 import os
 
 
 class LocalSQLProcessor(BasicSQLProcessor):
-    # read all csv files in datadir, and use file name as table name
-    def __init__(self, datadir: str):
+    def __init__(self):
         self.tables = {}
         self.schemas = {}
-        datafiles = os.listdir(datadir)
-        for file in datafiles:
-            if file.endswith('.csv'):
-                table_name = os.path.splitext(file)[0]
-                df = pd.read_csv(os.path.join(datadir, file))
-                self.tables[table_name] = df
-                print("load table " + file + "with columns:")
-                print(df.columns)
-                self.schemas[table_name] = self.__convert_schema(
-                    df, df.columns)
+
+    def load_table(self, table_name: str, schema_str: str, table_path: str):
```

Comment:
This loading function will also be used within the `ExternalSQLProcessor`. Maybe we could create a class named `DataAccessor` to deal with the data loading so that we can reuse it easily.